File: src/main/java/com/qtech/bubbles/common/cursor/BlankCursorRenderer.java
package com.qtech.bubbles.common.cursor;

import com.qtech.bubbles.common.GraphicsProcessor;
import com.qtech.bubbles.common.renderer.CursorRenderer;

import java.awt.*;

public class BlankCursorRenderer extends CursorRenderer {
    public BlankCursorRenderer() {
        super("blank_cursor");
    }

    @Override
    public void draw(GraphicsProcessor g) {

    }
}

File: src/main/kotlin/com/qtech/bubbles/common/cursor/BlankCursorRenderer.kt
package com.qtech.bubbles.common.cursor

import com.qtech.bubbles.common.renderer.CursorRenderer
import com.qtech.bubbles.graphics.GraphicsProcessor

class BlankCursorRenderer : CursorRenderer("blank_cursor") {
    override fun draw(g: GraphicsProcessor) {}
}